data/ecosystems/f/flow.toml

```diff
@@ -867,12 +814,6 @@ url = "https://github.com/brightamoi/flow-listings-viewer"
 [[repo]]
 url = "https://github.com/brightamoi/Flow-name-service"
 
-[[repo]]
-url = "https://github.com/brudfyi/it-scripts"
-
-[[repo]]
-url = "https://github.com/brudfyi/token-lockdrop"
```

Comment:
shouldn't this repo be part of Flow?

Reply:
This is a closed project

Reply:
wdym by "closed". We don't delete repos unless they became private. This is because we want to keep a full record of development on all ecosystems. Let's undo this deletion

Reply:
Done